runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -231,6 +235,40 @@ public void onNavigationButtonRemoved(final NavigationButtonRemoved event)
 		});
 	}
 
+	@Subscribe
+	public void onGameTick(final GameTick event)
+	{
+		if (!(client instanceof Client))
+		{
+			return;
+		}
+
+		if (lastName != null && !config.usernameInTitle())
```

Comment:
Why is `lastName != null` part of this check?

Reply:
To only update the frame title once